hcl2/transformer.py

```diff
@@ -116,7 +136,7 @@ def attribute(self, args: List) -> Dict:
         value = self.to_string_dollar(args[1])
 
         return {
-            key: value
+            key: value,
```

Comment:
Is the comma needed?

Reply:
it is just for auto formatting, without it will change the format to one line. I'm using `black` as my go to Python code formatter.